### PR TITLE
Updated Introduction to xarray, InferenceData, and NetCDF 

### PR DESCRIPTION
**Pull request by roshnaeem**
**Description:**
Updated the [Introduction to xarray, InferenceData, and NetCDF for ArviZ](https://arviz-devs.github.io/arviz/getting_started/XarrayforArviZ.html) part.

### CODE REVIEW

**Review by aloctavodia — APPROVED** [review]

File: doc/source/getting_started/XarrayforArviZ.ipynb [hunk 1]
```diff
@@ -12,17 +12,17 @@
    "cell_type": "markdown",
```
Comment (OriolAbril) [inline doc/source/getting_started/XarrayforArviZ.ipynb:12]: <p>I would not link to creating inference data here but do so below instead. I don't think people reading this explanation will expect to go to the creation tutorial at that point with that text.</p><p> </p><p>I would also add the general link to netcdf in the bullet points (same link as below in the netcdf section).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz/pull/1782/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (OriolAbril) [inline doc/source/getting_started/XarrayforArviZ.ipynb:12]: <p>I feel more natural to mention here that there are many ways to start an inferencedata, link to creating inferencedata and then saying here we'll load one example provided by the library for illustration purposes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz/pull/1782/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (OriolAbril) [inline doc/source/getting_started/XarrayforArviZ.ipynb:12]: <p>I think we should use netCDF as a general rule but treat it as a word so the <code>N</code>  is capitalized at the start of a sentence and so on. </p><p> </p><p>The manual anchor from which to link in the quickstart should be right above the netcdf title, with the <code>(anchor_id)=</code>  format.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz/pull/1782/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (OriolAbril) [inline doc/source/getting_started/XarrayforArviZ.ipynb:12]: <p>I would aim for link plus 1-2 sentences explaining what they'll find there. i.e. I am not completely sure what the basics of netCDF are, but the link directs me to the netcdf4-python library, a low level library to read/write netcdf files. The 2nd link however goes to xarray io page about read/write netcdf files directly as xarray objects. </p><p> </p><p>side note, the link to xarray page can be </p><p> </p><pre class="ql-syntax" spellcheck="false">{ref}<code>xarray:io.netcdf</code>
</pre><p> </p><p>see first sentence in https://github.com/pydata/xarray/blob/main/doc/user-guide/io.rst or use <code>sphobjinv</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz/pull/1782/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

**Review by OriolAbril — APPROVED** [review]

**Pull request merged**